posts/lte-examples.md

```diff
@@ -0,0 +1,491 @@
+There is an obvious forgetful functor to the non-strict category, which is denoted by `CHFPNG₁_to_CHFPNGₑₗ`:
+```lean
+example : CompHausFiltPseuNormGrp₁ ⥤ CompHausFiltPseuNormGrp :=
+CHFPNG₁_to_CHFPNGₑₗ
+
+example (X : CompHausFiltPseuNormGrp₁) :
+  (CHFPNG₁_to_CHFPNGₑₗ X : Type*) = X := rfl
+```
+
+## The associated condensed abelian group
+There is a natural functor from `CompHausFiltPseuNormGrp.{0}` to `Condensed.{0} Ab.{1}` which sends $M$ to the colimit $\bigcup_c M_c$.
+Here $M_c$ is viewed as a condensed *set* whose underlying presheaf is the representable presheaf associated to the topological space $M_c$.
+In LTE, we defined this functor in a more hands-on way.
+The functor itself is called `CompHausFiltPseuNormGrp.to_Condensed`:
+```lean
+example : CompHausFiltPseuNormGrp.{0} ⥤ Condensed.{0} Ab.{1} :=
+CompHausFiltPseuNormGrp.to_Condensed
+```
+and on objects it is defined as follows: 
+```lean
+example (X : CompHausFiltPseuNormGrp.{0}) (S : Profinite.{0}) :
+(Γ_ (CompHausFiltPseuNormGrp.to_Condensed X) S : Type 1) =
+(ulift.{1}  
+  { f : S → X | ∃ (c : ℝ≥0) (g : S → filtration X c), 
+    continuous g ∧ f = coe ∘ g }) := 
+rfl
+```
+
+Since Lean's type theory does not have cumulative universes, this definition involves a universe bump using `ulift`, in order to obtain an object of `Ab.{1}` as opposed to `Ab.{0}` (see the discussion above).
+Putting that aside, the sections of the condensed abelian group associated to a CHFPNG $X$ over a profinite set $S$ is the set of functions $f : S \to M$ which factor through a continuous map $g : S \to M_c$ for some $c$.
+
+The group structure on this set of sections is the obivous one.
+```lean
+example (X : CompHausFiltPseuNormGrp.{0}) (S : Profinite.{0})
+  (f g : Γ_ (CompHausFiltPseuNormGrp.to_Condensed X) S) (s : S) :
+  (f + g) s = f s + g s := rfl
+```
+
+## $p$-Radon measures
+Fix a real number $p$ satisfying $0 < p \le 1$.
+We will now discuss the relationship between $\mathcal{M}\_{p}(S)$ and the space of $p$-Radon measures.
+First we address the question of actually defining the space of $p$-Radon measures.
+Given any `S : Profinite.{0}`, and $p$ as above, we define `S.Radon_png p`, an object of `CompHausFiltPseuNormGrp₁`, as follows.
+
+As a set, it consists of all continuous linear maps $\mu : C(S,\mathbb{R}) \to \mathbb{R}$ such that there exists some $C \in \mathbb{R}_{\geq 0}$, where for any partition $S = V_1 \cup \cdots \cup V_n$ into disjoint clopen sets, letting $I_i \in C(S,\mathbb{R})$ denote the (continuous) indicator function of $V_i$, one has 
+$$ \sum_i |\mu(I_i)|^p \le C. $$
+Since $S$ is compact, Hausdorff and totally disconnected, this does indeed agree with the usual space of signed $p$-Radon measures (which reduces to the usual notion of a signed Radon measure when $p = 1$). 
```

Comment:
Note to self: this identification only yields a homeomorphism on the subset of measures bounded by a given `c`.